src/packages/@app/pages/Collections/CollectionsPage.svelte

```diff
@@ -69,6 +70,11 @@
   let splitter: HTMLElement | null;
   let isExposeSaveAsRequest: boolean = false;
   let isAppVersionVisible = true;
+  let isGuestUser = false;
+
+  isGuestUserActive.subscribe((value) => {
+    isGuestUser = value;
+  });
 
```

Comment:
why we are using store here, can we make a query to db, and identify weather guest user exist or not.

Reply:
through this we can restrict ourself to hit DB again and again, and their are also some places where we don't have view model so it better make sense to save it in store and use.

Reply:
hitting local db is okay, its a very light operation, @gc-codes i think its  better to handle it with db directly without using "store" as a middleware, that makes an overhead.